index.html

```diff
@@ -6,7 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
     <script src="https://kit.fontawesome.com/34795ce01c.js" crossorigin="anonymous"></script>
-    <title>Portfolio</title>
+    <title>Dhiraj | Portfolio</title>
+    <link rel="icon" type="image/png" sizes="16x16" href="images/Log-head.png">
```

Comment:
Good usage and explanation offered on the alt images. This is very important for your websites Accessibilty